packages/swapper/src/swappers/zrx/utils/applyAxiosRetry.ts

```diff
@@ -0,0 +1,18 @@
+import { AxiosInstance, AxiosRequestConfig } from 'axios'
+import { attach, RetryConfig } from 'retry-axios'
+
+type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>
```

Comment:
did we reinvent a wheel here or is this doing something different?

https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys